### PR TITLE
Fixing popover content for cartesian and Vertical bar chart

### PR DESCRIPTION
**Pull request by srmukher**
Fixing popover content for cartesian and Vertical bar chart.

### CODE REVIEW

File: packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx [hunk 1]
```diff
@@ -193,20 +200,9 @@ const PopoverComponent: React.FunctionComponent<IPopoverComponentProps> = React.
...
-                      borderLeft: `4px solid ${xValue.color}`,
-                      marginTop: props.XValue ? '13px' : 'unset',
-                    }
-                  : {}),
-              }}
             >
               <div className={classes.calloutlegendText}> {xValue.legend}</div>
-              <div
-                className={classes.calloutContentY}
-                style={{ color: props.color ? props.color : tokens.colorNeutralForeground1 }}
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx:208]: where are you setting the colors for the y callout content

File: packages/react-components/react-charts-preview/library/src/components/VerticalBarChart/VerticalBarChart.tsx [hunk 2]
```diff
@@ -267,15 +268,18 @@ export const VerticalBarChart: React.FunctionComponent<IVerticalBarChartProps> =
...
-          key={index}
-          Legend={item.legend}
-          {...(index === 0 && { XValue: `${hoverXValue || item.data}` })}
-          color={item.color}
-          YValue={item.data || item.y}
-          culture={props.culture}
-        /> */
+        <>
+          <Popover
+            culture={props.culture!}
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/library/src/components/VerticalBarChart/VerticalBarChart.tsx:273]: use 'en-us' if culture is not set

File: packages/react-components/react-charts-preview/library/package.json [hunk 3]
```diff
@@ -37,6 +37,7 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@fluentui/react-button": "^9.3.89",
```
Comment (AtishayMsft) [inline packages/react-components/react-charts-preview/library/package.json:40]: ensure that the build is passing before merging your changes

Comment (srmukher) [reply packages/react-components/react-charts-preview/library/package.json:40]: yes build is passing, checked

File: packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx (hunk 1 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx:208]: colors need not be set for calloutContentY, as per v8

File: packages/react-components/react-charts-preview/library/src/components/VerticalBarChart/VerticalBarChart.tsx (hunk 2 above)
Comment (srmukher) [reply packages/react-components/react-charts-preview/library/src/components/VerticalBarChart/VerticalBarChart.tsx]: done

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**